assets/javascript/app.js

```diff
@@ -0,0 +1,157 @@
+        answer: "D"
+    },
+    {
+        question: "Which planet has the most gravity?",
+        choices: ["A: Venus", "B: Jupiter", "C: Mars", "D: Saturn"],
+        answer: "B"
+    },
+    {
+        question: "Which country produces the most coffee in the world?",
+        choices: ["A: Colombia", "B: Mexico", "C: Brazil", "D: Chile"],
+        answer: "C"
+    },
+    {
+        question: "In which body part can you find the femur?",
+        choices: ["A: Leg", "B: Arm", "C: Back", "D: Stomach"],
+        answer: "A"
+    },
+    {
+        question: "Which country invented tea?",
+        choices: ["A: America", "B: China", "C: Germany", "D: Greece"],
+        answer: "B"
+    },
+    {
+        question: "How many Lord of the Rings films are there?",
+        choices: ["A: 1", "B: 4", "C: 3", "D: 7"],
+        answer: "C"
+    },
+    {
+        question: "In what year was the first episode of South Park aired?",
+        choices: ["A: 1997", "B: 1990", "C: 1994", "D: 2000"],
+        answer: "A"
+    }
+]
+var i = 0;
+var triviaQuestion = triviaQuestions[i];
+var time = 30;
+var timer;
+var numberofQuestions = triviaQuestions.length;
+var correctQuestions = 0;
+
+function playGame() {
+    runTimer();
+    $('#startGame').css('display', 'none');
+    if (i === triviaQuestions.length) {
+        clearInterval(timer);
+        $('#timer').css("display", "none");
+        $('#totalscore').css("display", "block").append(`<h1>YOU SCORED: ${correctQuestions}/${numberofQuestions}</h1>`)
+        if (correctQuestions <= 6) {
+            var failQueryURL = "https://api.giphy.com/v1/gifs/random?api_key=BkaUZZWcFij6J7AoQj3WtPb1R2p9O6V9&tag=failed";
+            $.ajax(failQueryURL).then(function (response) {
+                var failImageUrl = response.data.fixed_width_small_url;
+                var failImage = $("<img>");
+                failImage.attr("src", failImageUrl);
+                failImage.attr("alt", "failed trivia image");
+                $("#totalscore").append(failImage);
+            });
+        } else {
+            var congratsQueryURL = "https://api.giphy.com/v1/gifs/random?api_key=BkaUZZWcFij6J7AoQj3WtPb1R2p9O6V9&tag=congrats";
+            $.ajax(congratsQueryURL).then(function (response) {
+                var congratsImageUrl = response.data.fixed_width_small_url;
+                var congratsImage = $("<img>");
+                congratsImage.attr("src", congratsImageUrl);
+                congratsImage.attr("alt", "congrats trivia image");
+                $("#totalscore").append(congratsImage);
+            });
+        }
+    } else {
+        $('#question').append(`<h2>${triviaQuestion.question}</h2>`)
+        triviaQuestion.choices.forEach(function (choice) {
+            $(`<div id="${choice.charAt(0)}" class="choices"><h3>${choice}</h3></div>`).appendTo('#choices').one("click", function () {
+                if (choice.charAt(0) == triviaQuestion.answer) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='. (eqeqeq)](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555494)